thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-metric-dimension/template.hbs

```diff
@@ -11,6 +11,8 @@
     </label>
     {{rootcause-select-metric
       selectedUrn=baseUrn
+      metricEntities=entities
```

Comment:
I'd call this "entities" throughout. Most RCA components use that convention

Reply:
Good catch. I had it as `entities=entities` but changed it to test passing in only metrics only to the components vs having the component derive the metrics but seems better having it in the component for separation of concerns.

Reply:
reverted it back to `entities=entities`